modules/mono/godotsharp_dirs.cpp

```diff
@@ -216,13 +216,11 @@ class _GodotSharpDirs {
 #endif
 
 #ifdef OSX_ENABLED
-		if (!DirAccess::exists(data_mono_root_dir)) {
-			data_mono_etc_dir = exe_dir.plus_file("../Resources/GodotSharp/Mono/etc");
-			data_mono_lib_dir = exe_dir.plus_file("../Frameworks/GodotSharp/Mono/lib");
-		}
+		data_mono_etc_dir = exe_dir.plus_file("../Resources/GodotSharp/Mono/etc");
+		data_mono_lib_dir = exe_dir.plus_file("../Frameworks/GodotSharp/Mono/lib");
```

Comment:
I changed those two and `data_game_assemblies_dir` back as the `GodotSharp/` part had been removed, while it's still being used by the updated build scripts from @neikeq.